plugins/capistrano/capistrano.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+# Added `shipit` because `cap` is a reserved word. `cap` completion doesn't work.
```

Comment:
I think if you rename `_capistrano` to `_shipit` to fix the `shipit` completion, then a user can just do `alias cap=shipit` and the completion will work for them. However, that will break other uses of `cap` which are not themselves protected by doing `builtin cap` or `noalias` or `autoload -U` or something similar.